v1_1/messages.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package crowdnotifier_v1_1;
+
+message VenueType{}
+
+message QRCodeContent{
+  int32 version = 1;
+  bytes publicKey = 2;
+  string name = 3;
+  string location = 4;
+  string room = 5;
+  VenueType venueType = 6;
+  bytes notificationKey = 7;
+  bytes pEnt = 8;
+}
+
+message QRCodeTrace{
+  int32 version = 1;
+  bytes trSk = 2;
+  bytes trR2 = 3;
+  bytes pTrR1 = 4;
+  bytes pTrR2 = 5;
```

Comment:
For this implementation, no need to pack R2 twice and then check they are equal. Once is enough. (It is done this way to make the crypto-writeup easier, no need to actually store it twice.)

Reply:
simplified the QRcode.